.github/pull_request_template.md

```diff
@@ -1,8 +1,10 @@
+Attached to issue: #
```

Comment:
I was under the impression we wanted to add `https://github.com/DataBiosphere/azul/issues/<issue number>`, only adding `#number` doesn't have an effect in the linking.

Reply:
It actually does affect the linking. I tested this in this PR, but also check out this comment https://github.com/DataBiosphere/azul/issues/3219#issuecomment-877552166

Reply:
https://zenhub-users.slack.com/archives/CT6PG50JX/p1619694907124500?thread_ts=1619632116.117900&cid=CT6PG50JX using URLs for linking issues automatically to PRs.